src/components/sidebar/helpers/generate-product-landing-nav-items.ts

```diff
@@ -75,5 +83,6 @@ export const generateProductLandingSidebarNavData = (product: ProductData) => {
 		menuItems,
 		showFilterInput,
 		title,
+		visuallyHideTitle: true,
```

Comment:
A note here re: that we insert the 'themed' title would be helpful.

Reply:
💖 great call - added! ✍️